contracts/interfaces/dYdXSoloMargin.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.2;
+pragma experimental ABIEncoderV2;
+
+library Account {
+  enum Status {
```

Comment:
I need to configure the solhint settings to be stricter but we should use 4 spaces for indention.